### ISSUE TITLE
Trigger Type restriction

### ISSUE THREAD

**Issue by ekx77** [dahall/TaskScheduler#32]
I want to restrict trigger type selection in editor (actually I want only calendar triggers and don't want event, logon etc triggers). Is that possible?

Best regards 
Kirill

**Comment by dahall** [comment]
To be clear, you want to limit the type of trigger that can be created when clicking the New button or you want to disable the editing of certain types of triggers that are already added? Neither are possible right now, but it would be an easy addition.

**Comment by ekx77** [comment]
First.
But if user edit existing trigger, he should be able to change it's type only to limited set.

**Comment by dahall** [comment]
Would you expect to set the restrictions from TaskEditorDialog, TaskPropertiesControl or TriggerEditDialog?

**Comment by ekx77** [comment]
TaskEditorDialog would be best.

**Comment by dahall** [comment]
[TaskScheduler.zip](https://github.com/dahall/TaskScheduler/files/1462952/TaskScheduler.zip)
[TaskSchedulerEditor.zip](https://github.com/dahall/TaskScheduler/files/1462953/TaskSchedulerEditor.zip)
The two files linked here include a prerelease of v2.7 which includes the enhancements you've requested. Please test and confirm that the changes meet your requirements.

Usage:
```c#
var editorForm = new TaskEditDialog(taskService, taskDefinition, true, true)
{
	AvailableTabs = AvailableTaskTabs.All,
	AvailableActions = AvailableActions.ComHandler | AvailableActions.Execute,
	AvailableTriggers = AvailableTriggers.Daily | AvailableTriggers.Weekly,
	TaskName = "Test"
}
```

**Comment by ekx77** [comment]
Yes it works perfectly! Exactly the way I wanted, thank you very much!

**Comment by dahall** [comment]
Released in v2.7.0 today. You can find example code on the Triggers page of the wiki.
